xpub.go

```diff
@@ -77,6 +77,10 @@ func (xpub *xpubSocket) SetOption(name string, value interface{}) error {
 	return xpub.sck.SetOption(name, value)
 }
 
+func (xpub *xpubSocket) Topics() []string {
```

Comment:
we could perhaps add a test for that new method?

Reply:
sure added a test.

Reply:
And https://github.com/go-zeromq/license/pull/6

Reply:
unfortunately, this breaks the compatibility test w/ ZeroMQ (C++).
we might have to create a dedicated test for Topics. (implementing `Topics` for the C compat socket might be overkill)

Reply:
How do I run that compatibility test? I've run with -tags czmq4 and without. I didn't see any other +build flags in the project.

Reply:
yes, `czmq4` is the one.
I must admit I didn't run your PR locally, and just relied on the output of Travis:

https://api.travis-ci.org/v3/job/730419187/log.txt
```
=== CONT  TestXPubSub/ipc-cxpub-sub
panic: interface conversion: *zmq4.csocket is not zmq4.Topics: missing method Topics

goroutine 672 [running]:
github.com/go-zeromq/zmq4_test.TestXPubSub.func1.1(0xc000182f50, 0x0)
	/home/travis/gopath/src/github.com/go-zeromq/zmq4/zmq4_xpubsub_test.go:116 +0x233
golang.org/x/sync/errgroup.(*Group).Go.func1(0xc00036c7e0, 0xc000362a90)
	/home/travis/gopath/pkg/mod/golang.org/x/sync@v0.0.0-20190423024810-112230192c58/errgroup/errgroup.go:57 +0x86
created by golang.org/x/sync/errgroup.(*Group).Go
	/home/travis/gopath/pkg/mod/golang.org/x/sync@v0.0.0-20190423024810-112230192c58/errgroup/errgroup.go:54 +0x74
FAIL	github.com/go-zeromq/zmq4	18.611s
FAIL
travis_time:end:18b75166:start=1601079246730141654,finish=1601079273566725478,duration=26836583824,event=script
[0K[31;1mThe command "./.ci/code-coverage.sh" exited with 1.[0m

```